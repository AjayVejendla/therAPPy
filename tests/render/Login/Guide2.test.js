import React from "react";
import Guide2 from "../../../screens/Login/Guide2";
import renderer from "react-test-renderer";

test("Guide2 render test", () => {
  const tree = renderer.create(<Guide2 />).toJSON();
  expect(tree).toMatchSnapshot();
});
