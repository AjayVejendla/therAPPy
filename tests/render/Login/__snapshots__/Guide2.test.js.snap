// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guide2 render test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f2e9e1",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#7f58ff",
        "fontSize": 50,
        "fontWeight": "bold",
        "marginBottom": 40,
      }
    }
  >
    therAPPy
  </Text>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 18,
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    The way this app works is that you sign in and record how you are doing today. We will check how you feel today with past data and come up with some activities you can do to improve your mental health status today.
  </Text>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 18,
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    We will come up with these recommendations by checking your data from this app and events that are coming up in your area.
  </Text>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-evenly",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "marginLeft": 90,
          "marginTop": 25,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#7f58ff",
            "borderRadius": 10,
            "height": 40,
            "justifyContent": "center",
            "opacity": 1,
            "width": "45%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "black",
              "fontSize": 18,
              "marginLeft": 10,
              "marginRight": 10,
            }
          }
        >
          Back!
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "marginTop": 25,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#7f58ff",
            "borderRadius": 10,
            "height": 40,
            "justifyContent": "center",
            "opacity": 1,
            "width": "45%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "black",
              "fontSize": 18,
              "marginLeft": 10,
              "marginRight": 10,
            }
          }
        >
          Next!
        </Text>
      </View>
    </View>
  </View>
</View>
`;
